src/test/resources/good/ManyDispatchMethod.java

```diff
@@ -0,0 +1,31 @@
+package test;
+
+import com.bnorm.auto.dispatch.AutoDispatch;
+
+public class ManyDispatchMethod {
+
+  @DescriptionMulti(Age.Young)
+  public String descriptionYoung(Person person) {
+    return "You're so young!";
+  }
+
+  @DescriptionMulti({Age.Old})
+  public String descriptionOld(Person person) {
+    return "You're quite old...";
+  }
+
+  @DescriptionMulti({Age.Young, Age.Old, Age.VeryOld})
+  public String descriptionDefault(Person person) {
```

Comment:
Can you define the annotation with `@MultiDescription(Age.values())`?  It's not going to have the greatest performance but it would at least get you the behavior if you added more values later.